config/target.json

```diff
@@ -0,0 +1,5 @@
+{
+  "generate_icicle": false,
+  "openstack_image_format": "qcow2",
+  "rhevm_image_format": "qcow2"
```

Comment:
With the image format settings in this file now, can we remove them from imagefactory.conf?
https://github.com/ManageIQ/manageiq-appliance-build/blob/master/config/imagefactory.conf#L8-L9